app/models/user.rb

```diff
@@ -171,6 +171,10 @@ def is_test?
     !!is_test
   end
 
+  def opt_out_of_cookies?
+    !!opt_out_of_cookies
+  end
+
```

Comment:
I think defining this is not necessary, actually, because Rails essentially detects that the DB column is a boolean and then it automagically responds to the method with the same but with a question mark at the end. In this case, `opt_out_of_cookies?`. Yay, Rails magic! 😅

Reply:
I originally did not define it and the admin update did not work. However, I retested it without the method and the field updated. It is removed.